docs-website/docusaurus.config.js

```diff
@@ -4,8 +4,8 @@ const isSaas = process.env.DOCUSAURUS_IS_SAAS === "true";
 module.exports = {
   title: process.env.DOCUSAURUS_CONFIG_TITLE || "DataHub",
   tagline: "A Metadata Platform for the Modern Data Stack",
-  url: process.env.DOCUSAURUS_CONFIG_URL || "https://datahubproject.io",
-  baseUrl: process.env.DOCUSAURUS_CONFIG_BASE_URL || "/",
```

Comment:
pretty cool that this is all it takes to deploy a "demo" instance!

it might be possible to make it even simpler by setting `DOCUSAURUS_CONFIG_URL` and `DOCUSAURUS_CONFIG_BASE_URL` in the github actions yml